updatecli/policies/open-release-pr/updatecli.d/open-release-pr.yaml

```diff
@@ -0,0 +1,119 @@
+name: Open PR that prepares the policy for release
+pipelineid: "{{ .pipelineid }}"
+version: v0.97.0
+
+sources:
+  newestGitHubDraftReleaseVersion:
+    name: Extract version from newest GitHub draft release from release drafter
+    kind: githubrelease
+    spec:
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repo }}"
+      token: "{{ requiredEnv .github.token }}"
+      key: "title"
+      typefilter:
+        # We specifically only want the newest draft release, created with
+        # release-drafter, as its title has the prospective semver that we want
+        # to tag
+        draft: true
+      versionfilter:
+        kind: latest
+    transformers:
+      - trimprefix: "v"
+
+  newestReleaseTagVersion:
+    name: Extract newest tag from newest GitHub release
+    # used for displaying a gh compare URL in the open PR
+    # TODO continue on fail (we may not have a tag)
+    kind: githubrelease
+    spec:
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repo }}"
+      token: "{{ requiredEnv .github.token }}"
+      key: "tagname"
+      versionfilter:
+        kind: latest
+
+conditions:
+  checkCargoTomlExists:
+    name: Check if Cargo.toml exists
+    scmid: "policy-repo"
+    kind: file
+    disablesourceinput: true
+    spec:
+      file: Cargo.toml
+
+targets:
+  updateMetadataAnnotationVersion:
+    name: Update metadata.yml with new version
+    kind: yaml
+    scmid: "policy-repo"
+    sourceid: newestGitHubDraftReleaseVersion
+    disableconditions: true
+    spec:
+      searchpattern: true
+      file: "metadata.y*ml"
+      key: "$.annotations.'io.kubewarden.policy.version'"
+
+  updateCargoTomlVersion:
+    name: Update Cargo.toml with new version
+    scmid: "policy-repo"
+    dependson:
+      - condition#checkCargoTomlExists
+    sourceid: newestGitHubDraftReleaseVersion
+    kind: shell
+    spec:
+      shell: /usr/bin/bash
+      command: cargo set-version
```

Comment:
keep in mind this does not update the contents of `Cargo.lock`

Reply:
it does here!